CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.0 (2019-02-22)
+
+- Added ability to verify email addresses and URLs statically. [#9](https://github.com/blackbaud/skyux-validation/pull/9)
```

Comment:
This is probably fine, but thought I'd ask - a consumer might not know why this is useful. Should it be something like `...to verify email addresses and URLs outside of a formControl` or something?

Reply:
I think we should leave it. Technically you could use the static version in a form control so I don't want to confuse that.